mlrun/db/httpdb.py

```diff
@@ -1509,7 +1531,9 @@ def list_entities(
         return resp.json()["entities"]
 
     @staticmethod
-    def _generate_partition_by_params(partition_by, rows_per_partition, sort_by, order):
+    def _generate_partition_by_params(
+        partition_by_cls, partition_by, rows_per_partition, sort_by, order
+    ):
         if isinstance(partition_by, schemas.FeatureStorePartitionByField):
```

Comment:
```suggestion
        if isinstance(partition_by, partition_by_cls):
```
Was this what you meant to do? Maybe also assert if this is wrong?

Reply:
Yes, don't think it's needed